challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,11 @@
+const BinarySearch = (sortedArray, searchKey) => {
+  for(let i = 0; i < sortedArray.length; i++){
```

Comment:
function search(arr, searchKey) {
  for(let i = 0; i < arr.length; i++) {
    if(arr[i] === searchKey) return i
  }

  return -1
}